open-company/open-company-web
# Fix cut mentions

Card: https://trello.com/c/bmNPAKCT

To test:
- New post
- expand to fullscreen
- add title
- add abstract
- add a line of body
- try to mention someone
- [ ] do you see the popup? Good
- now add some more lines until you reach the bottom of the page
- try to mention someone again
- [ ] can you scroll the page to see the whole list of mention's user? Good
- [ ] make sure mobile body still works and is not broken
